.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "0.2.0",
+    "configurations": [
```

Comment:
Can we have two configurations? One for an integrated terminal and one for external?

Reply:
fixed.